include/boost/system/detail/error_code.ipp

```diff
@@ -388,7 +389,7 @@ namespace
             buf.resize(retval);
             break;
         }
-        else if ( ::GetLastError() != ERROR_INSUFFICIENT_BUFFER )
+        else if (boost::detail::winapi::GetLastError() != ERROR_INSUFFICIENT_BUFFER)
```

Comment:
Error codes should also be converted to Boost.WinAPI: `ERROR_INSUFFICIENT_BUFFER` => `boost::detail::winapi::ERROR_INSUFFICIENT_BUFFER_`.